core/dbt/contracts/graph/nodes.py

```diff
@@ -454,7 +454,7 @@ def resource_class(cls) -> Type[HookNodeResource]:
 
 @dataclass
 class ModelNode(ModelResource, CompiledNode):
-    batches: Optional[List[BatchType]] = None
+    batch_info: Optional[BatchResults] = None
```

Comment:
Will this cause a loading error for a previous manifest?

Reply:
This has no effect on the written manifest, which is intentional. That is the case because these do not exist on the artifact class for the object